### ISSUE TITLE
Complex modification not working on Sonoma

### ISSUE THREAD

**Issue by obiii** [pqrs-org/Karabiner-Elements#3741]
macOs: Sonoma
Karabiner: 14.13.0

Hi,

I am trying to use some complex modifications (one at a time) here: https://ke-complex-modifications.pqrs.org/#swedish_mx_keys_programming

I have tried everything but it doesnt work at all. Can someone please help?

So far I tried only one modification using "add your own rule":
```

{
      "description": "Swedish keyboard, Right command + 7,0 => curly-brackets {}",
      "manipulators": [
        {
          "type": "basic",
          "from": {
            "key_code": "7",
            "modifiers": {
              "mandatory": [
                "right_command"
              ]
            }
          },
          "to": [
            {
              "key_code": "8",
              "modifiers": [
                "left_option",
                "left_shift"
              ]
            }
          ]
        },
        {
          "type": "basic",
          "from": {
            "key_code": "0",
            "modifiers": {
              "mandatory": [
                "right_command"
              ]
            }
          },
          "to": [
            {
              "key_code": "9",
              "modifiers": [
                "left_option",
                "left_shift"
              ]
            }
          ]
        }
      ]
    }
```

**Comment by colingoodman** [comment]
I am also having this issue with complex modifications in general on Sonoma.

**Comment by bobmagicii** [comment]
yes, they flat out do not import at all. no matter how basic you make it, or how much you cut out of the json as suggested in other threads.

![image](https://github.com/pqrs-org/Karabiner-Elements/assets/881944/487658b7-f96a-4d82-b127-f90972004648)

**Comment by Percy-Deng** [comment]
Same issue here.

**Comment by snevs** [comment]
> yes, they flat out do not import at all. no matter how basic you make it, or how much you cut out of the json as suggested in other threads. doing it manually by copying the json exposes this manipulator errors that has a bug report already that has gone stale (#3117). pick and choose any of them from the website.
> 
> if you dont try to do it manually with the json, the experience is "ok nothing happened"

Any workarounds for this? Importing from repositories work, but I'd rather write my own JSONs.

**Comment by bobmagicii** [comment]
> Any workarounds for this? Importing from repositories work, but I'd rather write my own JSONs.

no i had completely giving up on karabiner for the task i was trying to perform. and the task (which was making mac os less offensive to my windows gamer habits)

**Comment by nejmeijer** [comment]
The failing input behaviour could be a bug in KE.

**Comment by felipemaion** [comment]
> The failing input behaviour could be a bug in KE.

@nejmeijer indeed this seems to be the case, check out this reply I gave on another opened issue: https://github.com/pqrs-org/Karabiner-Elements/issues/3117#issuecomment-2833620711 

I was able to make it work when KE is closed (if it is set to run prior to login)
